### PR TITLE
Show stability pool to liquity module

### PR DESCRIPTION
**Pull request by lukicenturi**
Frontend part of #4954

## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.

image

### CODE REVIEW

**Comment by yabirgb** [comment]
Different things I noticed:

- The filter by account doesn't detect all the accounts in the stability pool, only the ones with LQTY staked.
- `Pools` is a confusing name. I would write there `Stability pool deposits` or something similar
- Also having a breakdown per account there would be much better. RIght now you see the sum of all of them
- instead of ETH I would use `ETH gains` and same for `LQTY`
- If I have multiple accounts I would expect a breakdown per account and then one summary aggregating all the values

**Comment by lukicenturi** [comment]
UI Updated @yabirgb

File: docs/changelog.rst [hunk 1]
```diff
@@ -3,6 +3,7 @@ Changelog
 =========
 
 * :release:`1.26.1 <2022-11-04>`
+* :feature:`5102` Show stability pool to liquity module.
```
Comment (yabirgb) [inline docs/changelog.rst:6]: This should change but we can do in other PR. 

```suggestion
* :feature:`5102` Users will be able to track their LUSD and rewards in the Liquity's stability pool
```

**Review by yabirgb — APPROVED** [review]
As discussed in the stand up we are merging it as functionality is already fine

File: frontend/app/src/premium/register-components.ts [hunk 2]
```diff
@@ -129,6 +130,8 @@ export function registerComponents() {
   Vue.component('GraphTooltipWrapper', GraphTooltipWrapper);
   // Version 19 - 1.26
   Vue.component('LpPoolIcon', LpPoolIcon);
+  // Version 20 - 1.27
+  Vue.component('BadgeDisplay', BadgeDisplay);
```
Comment (kelsos) [inline frontend/app/src/premium/register-components.ts:134]: I will take care of it, but in order for this to properly work and not try to load we also have to do a bump here:

https://github.com/rotki/rotki/blob/a414eec6bbd51430dc01fd982b2805f7af86325e/frontend/app/src/premium/setup-interface.ts#L101

**Pull request merged**